tests/testthat/test-scm.R

```diff
@@ -1,9 +1,10 @@
-context("SCM")
+context("SCM-FF16")
 
-strategy_types <- get_list_of_strategy_types()
+strategy_types <- get_list_of_strategy_types()["FF16"]
```

Comment:
Does this mean we can ditch the K93 case below?

Reply:
Why not test them all with separate context labels as in Canopy?

Reply:
We could. I decided to restrict to FF16 so that tests run quickly. 

Can add FF16r and K93 back once we figure out why they’re running so much slower. 

They’re still run as part of the benchmarking.